packages/server/lambdas/alfresco/fileRequestBuilder/alfrescoRequestBuilder.ts

```diff
@@ -36,6 +41,15 @@ export class AlfrescoFileRequestBuilder {
     } as unknown as RequestInit;
     return options;
   }
+  public async updateRequestBuilder(event: ALBEvent, headers: HeadersInit) {
+    const buffer = base64ToBuffer(event.body as string);
+    const options = {
+      method: 'PUT',
+      body: buffer,
+      headers: headers,
+    } as unknown as RequestInit;
```

Comment:
Curious about the unknown here. Got anything to share?

Reply:
I don't know reason for this either 😕 Must be unnecessary copypaste mistake. I'll remove this before merge.

Reply:
This is actually fixed in #163